### PR TITLE
Improve debian build script

### PR DESCRIPTION
**Pull request by Frost**
The build script for debian was broken. It had shell script bugs (`$()` instead of `${}`, for instance) and tried to run `debuild` in the wrong directory.

I fixed that, and also made it do a full build (`debuild -F`), which means that it will produce both a `.deb` and a `.dsc` file.

Building a source package will help immensely building the ubuntu packages, since they can then be built using the [`backportpackage`](http://manpages.ubuntu.com/manpages/oneiric/man1/backportpackage.1.html) command, instead of me having to build them manually.

### CODE REVIEW

**Comment by Frost** [comment]
I am wondering a bit however, whether or not to put the `.dsc` files somewhere under the `gh-pages` branch or not. Ideally, they should be signed, and the current build script doesn't seem to sign the debian packages at all (correct me if I'm wrong).

I could change the `debuild` line so that it would sign the packages as well, but @mike-burns mentioned something about him having some problems with that earlier, because he has multiple private GPG keys. That could however be solved by making sure to set the `DEBSIGN_KEYID` env var before running `./maint/release build deb rcm-*.tar.gz`.

File: maint/release.in [hunk 1]
```diff
@@ -115,7 +114,7 @@ release_push_tag() {
   git push origin --tags
 }
 
-release_clean_tag() {
+release_clean_tag() { :
```
Comment (mike-burns) [inline maint/release.in]: Can you put the `:` on its own line? That will help it stand out more.

**Comment by mike-burns** [comment]
If the `dsc` file is helpful for other maintainers, then it should go under the `gh-pages` branch too, and at least be mentioned in `DEVELOPERS.md` if not the README.

Feel free to turn signing on, and I'll have to set the right keyid in the right place.

**Comment by Frost** [comment]
A note about the `DEBSIGN_KEYID` variable btw: `debsign` [ignores env variables and instead sources `~/.devscripts`](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=444641)

**Comment by mike-burns** [comment]
Cool. Looks good to merge (I'd like to do that this week), though I'm going to s/folder/directory/ because my target audience doesn't use Windows.

**Comment by Frost** [comment]
Wow, that was embarassing. Anyway, I fixed it, so it's all about "directory" now. :)

**Comment by mike-burns** [comment]
Merged, thanks!

**Pull request closed without merging**